dts/arm/nordic/nrf52840.dtsi

```diff
@@ -175,6 +175,11 @@
 			label = "USBD";
 		};
 
+		usb_cdc: virtualcom {
+			compatible = "nordic,nrf-usbd";
+			label = "CDC_ACM";
```

Comment:
Doesn't "virtualcom" need a conventional name format?

Reply:
No idea what you mean by that, in fact I don't really understand DTS that well currently other than "its needed" for this particular problem because Kconfig doesn't let me set a console uart device name, only through DTS.

Reply:
Link to relevant dts files and snippets
https://github.com/zephyrproject-rtos/zephyr/blob/master/dts/x86/intel_curie.dtsi#L110
https://github.com/zephyrproject-rtos/zephyr/blob/master/samples/subsys/usb/console/arduino_101.overlay#L1

Reply:
I meant whether it needs at <name>@<address> labeling convention